pkg/daemon/daemon.go

```diff
@@ -428,7 +428,10 @@ func (d *Daemon) syncConsul() error {
 						meta[k] = v
 					}
 				}
-
+				serviceStatus := status
```

Comment:
Couldn't we just continue to use the `status` variable?

Reply:
the status variable is used for all the services, if we override it, it will affect other services.